engines/instance_verification/lib/instance_verification/providers/example.rb

```diff
@@ -55,4 +55,10 @@ def allowed_extension?
     # to be activated on SCC or not, i.e. LTSS in Azure Basic VM
     true
   end
+
+  def add_on
+    # method to check if a system has an add on product
+    # based on the system metadata
+    # and if so, it returns its real product class
```

Comment:
I cannot follow this comment, what does it mean that a system **has** an add on product, and what is the **real product class**? Also, add on product is not a term used in RMT, I assume you mean 'extension product'?

Reply:
a SUMA 5.X+ instance has a base product of MicroOS when the real product is SUMA